tests/system/action/speaker/test_delete.py

```diff
@@ -399,3 +401,71 @@ def test_with_paused_structure_level_speaker(self) -> None:
             },
         )
         assert sllos["remaining_time"] == 18
+
+    def add_coupled_countdown(self) -> None:
+        self.set_models(
+            {
+                "meeting/1": {
+                    "list_of_speakers_couple_countdown": True,
+                    "list_of_speakers_countdown_id": 75,
+                },
+                "projector_countdown/75": {
+                    "running": True,
+                    "default_time": 200,
+                    "countdown_time": floor(time()) + 100,
+                    "meeting_id": 1,
+                },
+                "speaker/890": {
+                    "begin_time": floor(time()) + 100,
+                },
+            }
+        )
+
+    def test_delete_update_countdown(self) -> None:
+        self.set_models(self.permission_test_models)
+        self.add_coupled_countdown()
+        response = self.request("speaker.delete", {"id": 890})
+        self.assert_status_code(response, 200)
+        countdown = self.get_model("projector_countdown/75")
+        assert countdown.get("running") is False
+        self.assertAlmostEqual(countdown["countdown_time"], 100, delta=200)
+
+    def test_delete_dont_update_countdown(self) -> None:
+        self.set_models(self.permission_test_models)
+        self.set_models(
+            {
+                "meeting/1": {
+                    "speaker_ids": [890, 891],
+                    "meeting_user_ids": [7, 8],
+                },
+                "user/8": {
+                    "username": "test_username2",
+                    "meeting_user_ids": [8],
+                    "is_active": True,
+                    "default_password": DEFAULT_PASSWORD,
+                    "password": self.auth.hash(DEFAULT_PASSWORD),
+                },
+                "meeting_user/8": {"meeting_id": 1, "user_id": 8, "speaker_ids": [891]},
+                "list_of_speakers/23": {"speaker_ids": [890, 891], "meeting_id": 1},
+                "speaker/891": {
+                    "meeting_user_id": 8,
+                    "list_of_speakers_id": 23,
+                    "meeting_id": 1,
+                },
+            }
+        )
+        self.add_coupled_countdown()
+        now = floor(time())
+        response = self.request("speaker.delete", {"id": 891})
+        self.assert_status_code(response, 200)
+        countdown = self.get_model("projector_countdown/75")
+        assert countdown.get("running") is True
+        self.assert_model_exists(
+            "projector_countdown/75",
+            {
+                "running": True,
+                "default_time": 200,
+                "meeting_id": 1,
+            },
+        )
+        self.assertAlmostEqual(countdown["countdown_time"], 100, delta=now)
```

Comment:
As 'countdown_time' has a timestamp while the countdown is running, we should compare to the timestamp 'now' with delta 100. Why don't you do this in the 'assert_model_exists'? Is it because this can have different results on different setups? Then how about passing `now` to 'add_coupled_countdown' to be using the same timestamp everywhere?

Reply:
Change the two parameters around and this PR should be perfect.